src/Http/LambdaResponse.php

```diff
@@ -71,4 +72,19 @@ public function toApiGatewayFormat(): array
             'body' => $this->body,
         ];
     }
+    public function toALBFormat(): array
+    {
+        // This is the format required by the ALB lambda integration https://aws.amazon.com/pt/blogs/networking-and-content-delivery/lambda-functions-as-targets-for-application-load-balancers/
+        $statusDescription = "$this->statusCode ";
+        $statusDescription .= $this->http_codes[$this->statusCode] ?? 'Unknown';
```

Comment:
I Agree.
While i am sure it was a must back then, i just tested and without even sending the statusDescription property it worked without problems. so i am removing it.
AWS must have updated its ALB so it would add it by default.